src/containers/dashboard/ApiBot.js

```diff
@@ -0,0 +1,227 @@
+        Cell: row => {
+          return (<FormattedDate
+            value={row.original.createdAt}
+            day='2-digit'
+            year='2-digit'
+            month='short'
+          />);
+        },
+        className: 'table_col_value table_bot_col_value',
+      },
+      {
+        minWidth: window.matchMedia('(max-width: 1028px)') ? 40 : 80,
+        Header: this.renderHeader(currentMode === ACTIVE_KEYS.value  ?
+          this.props.intl.messages['dashboard.activeAt'] :
+          this.props.intl.messages['dashboard.deletedAt']),
+        accessor: currentMode === ACTIVE_KEYS.value ?
+          'lastUsedAt' : 'deletedAt',
+        headerClassName: 'table_bot_header_value',
+        className: 'table_col_value upper table_bot_col_value',
+        Cell: row => {
+          return (<FormattedDate
+            value={currentMode === ACTIVE_KEYS.value ?
+              row.original.lastUsedAt :
+              row.original.deletedAt
+            }
+            day='2-digit'
+            year='2-digit'
+            month='short'
+          />);
+        },
+      },
+      {
+        Header: '',
+        minWidth: 24,
+        className: 'table_col_delete',
+        Cell: row => {
+          const canDeleteKey = true;
+          const onClick =  e => {
+            e.stopPropagation();
+            const currentRowKeyId = row.original._id;
+            if (is2FAEnable) {
+              this.props.showConfirmModal('dashboard.deleteConfirm', {},
+                () => {
+                  this.props.showTwoFactorAuthModal('', {}, async () => await this.props.deleteKey(currentRowKeyId));
+                }
+              );
+            } else {
+              this.props.showConfirmModal('dashboard.deleteConfirm', {}, async  () => await this.props.deleteKey(currentRowKeyId));
+            }
+          };
+          const className = classNames('delete_key_button', {can_delete_key: canDeleteKey});
+          return (<div className={className} onClick={onClick}/>);
+        },
+      }
+    ];
+  }
+
+  renderHeader = header => (
+    <div className="table_header_wrapper contract_header_wrapper">
+      <span className="table_header">{header}</span>
+    </div>
+  );
+
+  renderModeDropdown = () => {
+    const { currentMode } = this.state;
+    const modeList = [
+      ACTIVE_KEYS,
+      DELETED_KEYS
+    ];
+    return (
+      <div className="mode_select_wrapper">
+        <ExchangeSelect exchanges={ modeList }
+          exchange={ currentMode }
+          exchangesTitleClasses="mode_select_container"
+          defaultPlaceholder="Current Mode"
+          onChange={ mode => {
+            this.setState({currentMode: mode});
+          }}
+        />
+      </div>
+    );
+  }
+
+  renderContent() {
+    const { keysList, currentMode } = this.state;
+    const currentData = keysList.filter(listItem => currentMode === DELETED_KEYS.value ?
+      Object.prototype.hasOwnProperty.call(listItem, 'deletedAt') :
```

Comment:
Why not just `listItem.hasOwnProperty('deletedAt')` ?

Reply:
https://eslint.org/docs/rules/no-prototype-builtins

Reply:
No prototype builtins is recommended for prototypeless objects. Can keyList item be such object?